builtin/providers/google/resource_compute_instance_template.go

```diff
@@ -305,11 +305,9 @@ func buildNetworks(d *schema.ResourceData, meta interface{}) (error, []*compute.
 	for i := 0; i < networksCount; i++ {
 		prefix := fmt.Sprintf("network_interface.%d", i)
 
-		source := "global/networks/default"
+		source := "global/networks/"
```

Comment:
What about distinguishing between `network`, and a new field, `network_uri`. The idea would be to have Terraform do less "magic" and assemble the resources less opaquely.